src/App.tsx

```diff
@@ -1,24 +1,368 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  FormEvent,
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+import cn from 'classnames';
+import { getAllTodos } from './api/todos';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
+import { Filters, Errors } from './utils/enums';
 
-const USER_ID = 0;
+const USER_ID = 10306;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedFilter, setSelectedFilter] = useState('All');
```

Comment:
Consider using `useState<Filters>(Filters.All)` for the selectedFilter state so that its type is tied to the Filters enum.